Library/Homebrew/cask/cask.rb

```diff
@@ -78,6 +78,17 @@ def initialize(token, sourcefile_path: nil, source: nil, tap: nil, loaded_from_a
       end
     end
 
+    # An old name for the cask.
+    sig { returns(T::Array[String]) }
+    def old_tokens
+      @old_tokens ||= if tap
+        cask_renames = tap.cask_renames
+        cask_renames.flat_map { |k, v| (v == token) ? k : [] }
```

Comment:
```suggestion
        tap.cask_renames.select { |_, newtoken| newtoken == token }.keys
```

Nit: For clarity

This is just copied from the formula version.